### ISSUE TITLE
Error: Cannot read property '_storage' of undefined

### ISSUE THREAD

**Issue by mocsharp** [NasAmin/trx-parser#77]
Getting the following error, any idea?

https://github.com/mocsharp/clara-dicom-adapter/runs/1663835318?check_suite_focus=true

```
Run NasAmin/trx-parser@v0.0.3
  with:
    TRX_PATH: /home/runner/work/clara-dicom-adapter/clara-dicom-adapter/test-results
    REPO_TOKEN: ***
    IGNORE_FAILURE: false
Finding Trx files in: /home/runner/work/clara-dicom-adapter/clara-dicom-adapter/test-results
Files count: 10
Processing 7 trx files
Transforming file /home/runner/work/clara-dicom-adapter/clara-dicom-adapter/test-results/_3860a79a0dda_2021-01-07_16_58_06.trx
Error: Cannot read property '_storage' of undefined
```

**Comment by mocsharp** [comment]
Looks like it's failing due to one of the TRX files.  Please find the artifacts on https://github.com/mocsharp/clara-dicom-adapter/actions/runs/469640852 and look for `IntegrationCRD.trx` file.

Thanks.

**Comment by NasAmin** [comment]
@mocsharp Thanks for reporting this issue and providing the trx. I'll have a look and try to fix this over the weekend.

**Comment by NasAmin** [comment]
@mocsharp I have applied a fix. I have released the changes as `v0.0.5`. Please let me know if you see any further issues.
